addons/binding/org.openhab.binding.onewiregpio/src/main/java/org/openhab/binding/onewiregpio/OneWireGPIOBindingConstants.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.onewiregpio;
+
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+
+/**
+ * The {@link OneWireGPIOBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Anatol Ogorek - Initial contribution
+ */
+public class OneWireGPIOBindingConstants {
+
+    public static final String BINDING_ID = "onewiregpio";
+
+    public static final ThingTypeUID THING_TYPE = new ThingTypeUID(BINDING_ID, "sensor");
+
+    public static final String TEMPERATURE = "temperature";
+
+    /**
+     * The refresh time in seconds.
+     */
+    public static final String REFRESH_TIME = "refresh_time";
+
+    /**
+     * The default auto refresh time in seconds.
+     */
+    public static final Integer DEFAULT_REFRESH_TIME = Integer.valueOf(600);
```

Comment:
Is refreshing so expensive or why is it only done every 10 minutes?

Reply:
No, it is not expensive a lot. When there are many sensors it might take few seconds to read value. 
I thought there is no need to refresh more often.
I changed to 120s